pkg/service/v0/service.go

```diff
@@ -109,6 +109,18 @@ func (p Phoenix) getPayload() (payload []byte, err error) {
 	return
 }
 
+func singleJoiningSlash(a, b string) string {
+	aslash := strings.HasSuffix(a, "/")
+	bslash := strings.HasPrefix(b, "/")
+	switch {
+	case aslash && bslash:
+		return a + b[1:]
+	case !aslash && !bslash:
+		return a + "/" + b
+	}
+	return a + b
+}
```

Comment:
Woudln't `path.Join()` do what you intend here? Check this one out: https://play.golang.org/p/zYuldrHhFXd

Reply:
no, `path.Join` will change `http(s)://` to `http(s):/`

Reply:
then the `Path` attribute should not be called Path. Instead call it it should be called URL. `https` (the protocol) is not part of the path. Different issue :)

Reply:
Wow, I was just thinking `we would have to rename the key in phoenix then... wait, maybe it can be used as a path?!`. That works as well. Updating the PR...